<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Transformation2D</class>
 <widget class="QWidget" name="Transformation2D">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>500</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QSlider" name="translateX">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>80</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-250</number>
   </property>
   <property name="maximum">
    <number>250</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="translateY">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>130</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-250</number>
   </property>
   <property name="maximum">
    <number>250</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_translateX">
   <property name="geometry">
    <rect>
     <x>546</x>
     <y>80</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Translate X</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_translateY">
   <property name="geometry">
    <rect>
     <x>546</x>
     <y>130</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Translate Y</string>
   </property>
  </widget>
  <widget class="QSlider" name="scaleX">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>190</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>200</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="scaleY">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>240</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>200</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_translateX_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>180</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Scale X</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_translateX_3">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>230</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Scale Y</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reset">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>40</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QSlider" name="rotate">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>290</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
   <property name="value">
    <number>180</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_rotate">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>280</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Rotate</string>
   </property>
  </widget>
  <widget class="QSlider" name="sheareX">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>330</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-100</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="sheareY">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>370</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-100</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_sheareX">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>330</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Sheare X</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sheareY">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>370</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Sheare Y</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="interpolation">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>40</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Interpolation</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
